ci-operator/step-registry/openshift/upgrade/aws/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
```

Comment:
Symlink to `../OWNERS`?  I doubt we will need flavor-specific maintainer sets here, and your  current sets look like dups.